.github/workflows/build_canary.yml

```diff
@@ -13,61 +13,114 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Build and push operator image
-        uses: docker/build-push-action@v1.1.1
+      - name: Set up tags and refs
+        id: prep
+        run: |
+          SHA="sha-${GITHUB_SHA::8}"
+          echo ::set-output name=sha::${SHA}
+          echo ::set-output name=created::$(date -u +'%Y-%m-%dT%H:%M:%SZ')
```

Comment:
@khaosdoctor why not just use `${GITHUB_SHA::8}` in the below step where it's used?

Reply:
I was already going to use a different step to set some labels, so I kept everything in the same place. I can update it to remove the labels and the prepare step completely on this workflow

Reply:
seems like using the env var is simpler, so let's go with that

Reply:
So we just keep the way it is or should I use the `GITHUB_SHA` in the next steps?

Reply:
sorry I wasn't clear, @khaosdoctor! I meant, can you use `${GITHUB_SHA}`?

Reply:
Sure thing!